nextflow_schema.json

```diff
@@ -170,7 +164,8 @@
             "properties": {
                 "external_tools_meta": {
                     "type": "string",
-                    "description": "Specifies the path to the JSON file with meta information on external prediction tools."
+                    "description": "Specifies the path to the JSON file with meta information on external prediction tools.",
+                    "hidden": true
                 },
                 "netmhc_system": {
```

Comment:
Cannot remember right now why this is called `netmhc_system`. Wouldn't it be better to call it `netmhcpan_path` ?

Reply:
`netmhc_system` was to differentiate netmhc tools for mac or linux. There is already a `netmhcpan_path` param

Reply:
Ah it was showing the description of the `netmhc_path` item right below this one. That's why I got confused. All good. :)